sm2a/Makefile

```diff
@@ -48,7 +48,7 @@ sm2a-local-stop:
 
 sm2a-deploy:
 	@echo "Installing the deployment dependency"
-	pip install -r deploy_requirements.txt
```

Comment:
This path looks right though? Is it a cd in the deployment action?

Reply:
The command is supposed to run inside `sm2a` folder. deploy_requirements.txt lives in the root folder.